### PR TITLE
[deploy] build first when running

### LINKED ISSUE #780: [docs] deploy from source, but not found file or dir
**Describe the issue**

In https://datafuse.rs/overview/building-and-running/ , we can deploy from source: 

```
$ git clone https://github.com/datafuselabs/datafuse.git
$ cd datafuse
$ make setup
$ make run
```

but, if we do not build datafuse before running, will get some errors:

```
bash ./scripts/deploy/fusequery-standalone.sh
fuse-store: 未找到进程
fuse-query: 未找到进程
Start FuseStore...
Waiting on fuse-store 10 seconds...
nohup: 把输出追加到 'nohup.out'
nohup: 无法运行命令 'target/debug/fuse-store': 没有那个文件或目录
Start FuseQuery...
Waiting on fuse-query 10 seconds...
nohup: 把输出追加到 'nohup.out'
nohup: 无法运行命令 'target/debug/fuse-query': 没有那个文件或目录
make: *** [Makefile:9：run] 错误 124
```

### PR DESCRIPTION
**Pull request by PsiACE**
Signed-off-by: Chojan Shang <psiace@outlook.com>

I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

build first when running, to ensure that the deployment can be completed according to the documentation

## Changelog

- Build/Testing/CI

## Related Issues

Fixes #780

## Test Plan

Just test it locally

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,6 +6,7 @@ setup:
 	bash ./scripts/setup/dev_setup.sh
 
 run:
```
[the remark this answers is not in the document]
Comment (PsiACE) [reply Makefile:8]: ok, excuse me a moment.

[the remark this answers is not in the document]
Comment (BohuTANG) [reply Makefile:8]: :,

[the remark this answers is not in the document]
Comment (PsiACE) [reply Makefile]: now:
- `run` means `build native and run`
- `run-debug` means `build debug and run`
- `run-release` means `run from datafuse release`

**Comment by PsiACE** [comment]
Maybe we should also upload the compiled `fusestore` during release and modify the `./scripts/deploy/fusequery-standalone-from-release.sh`.

**Review by BohuTANG — APPROVED** [review]
LGTM

**Pull request merged**